src/Classes/Utils/Utils.ts

```diff
@@ -48,6 +91,14 @@ export class Utils {
         return destination;
     }
 
+    public static mixComponents(comp1: number, comp2: number, weight1: number, weight2: number): number {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public static method not allowed after declaration of private static method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)